Signal/src/ViewControllers/AttachmentApprovalViewController.swift

```diff
@@ -15,21 +15,14 @@ class AttachmentApprovalViewController: OWSViewController, OWSAudioAttachmentPla
 
     var successCompletion : (() -> Void)?
 
-    var videoPlayer: MPMoviePlayerController?
-
-    var audioPlayer: OWSAudioAttachmentPlayer?
-    var audioStatusLabel: UILabel?
-    var audioPlayButton: UIButton?
-    var isAudioPlayingFlag = false
-    var isAudioPaused = false
-    var audioProgressSeconds: CGFloat = 0
-    var audioDurationSeconds: CGFloat = 0
+    let mediaMessageView: MediaMessageView
```

Comment:
I pulled out `MediaMessageView` from the attachment approval view so that we can use it to easily and consistently display attachments in multiple views.